source/_components/emulated_hue.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: page
+title: "Emulated Hue Bridge"
+description: "Emulated Hue Bridge"
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+The `emulated_hue` component provides a virtual Philips Hue bridge, written entirely in software,
+that allows services that work with the Hue API to interact with Home Assistant
+entities. The driving use case behind this functionality is to allow Home
+Assistant to work with an Amazon Echo with no set up cost outside of
+configuration changes.
+
+To enable the emulated Hue bridge, add the following to your `configuration.yaml`
+file:
+
+```yaml
+# Example configuration.yaml entry
+emulated_hue:
+  host_ip: 192.168.1.186
+  listen_port: 8300
+  off_maps_to_on_domains:
+    - script
+    - scene
+  expose_by_default: true
+  exposed_domains:
+    - light
+
+# Example customization
```

Comment:
Let's break the customize block into its own section which will go under the Configuration variables section.